components/mdc/Datatable/DatatableCheckbox.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+import { generateRandomID } from '../../../random'
+import { createEventDispatcher, onMount } from 'svelte'
+
+export let disabled = false
+
+let checked
+
+const inputID = generateRandomID('checkbox-')
+
+onMount(() => dispatch('mounted'))
+
+$: checked, handleChange()
+
+const dispatch = createEventDispatcher()
+
+const handleChange = () => {
+  dispatch('change')
+}
+</script>
+
+<td class="mdc-data-table__cell mdc-data-table__cell--checkbox" on:click>
+  <div class="mdc-checkbox mdc-data-table__row-checkbox">
+    <input type="checkbox" class="mdc-checkbox__native-control" aria-labelledby={inputID} {disabled} bind:checked />
```

Comment:
This aria-labelledby seems like it's referring to some other element with that ID, but that `inputID` isn't used anywhere else. I might be missing something, but it seems like something is missing.

Reply:
You are correct. It should be the same as `Datatable.Data.Row` `rowId` so I will add a slot prop to expose `rowId` and we can use the let directive to bring it in here as a prop.